gui-libs/hyprcursor/hyprcursor-0.1.12.ebuild

```diff
@@ -7,11 +7,17 @@ inherit cmake
 
 DESCRIPTION="The hyprland cursor format, library and utilities"
 HOMEPAGE="https://github.com/hyprwm/hyprcursor"
-SRC_URI="https://github.com/hyprwm/hyprcursor/archive/v${PV}.tar.gz -> ${P}.gh.tar.gz"
+
+if [[ "${PV}" = *9999 ]]; then
```

Comment:
nit:

```suggestion
if [[ ${PV} == *9999* ]]; then
```

Reply:
Thanks, I have changed it in the release and live ebuild.